compat-data-schema.md

```diff
@@ -0,0 +1,365 @@
+# The browser-compat-data JSON format
```

Comment:
I think this doc could do with a copy-edit, but that can be done in a follow-up.